### PR TITLE
little fixes

### PR DESCRIPTION
**Pull request by Mte90**
Fix on error showed on #16

### CODE REVIEW

File: js/timelines.js [hunk 1]
```diff
@@ -421,7 +420,12 @@ function showListDetails(newHash, excludeUS){
         });
       }
     });
-    var timedata = masterBugTable.lists[list].timedata, openBugCount = masterBugTable.lists[list].counts.open, resolvedBugCount=masterBugTable.lists[list].counts.resolved;
+    var timedata = [], openBugCount = '', resolvedBugCount = '';
+    if(typeof masterBugTable.lists[list].counts !== 'undefined') {
```
Comment (hallvors) [inline js/timelines.js]: Maybe you have to check `typeof masterBugTable.lists[list]` too?

Comment (hallvors) [inline js/timelines.js]: (in case 'list' is wrong - not sure if this happens)

Comment (Mte90) [inline js/timelines.js]: the error is that `counts` that is undefined so I guess that happen with both.

**Comment by Mte90** [comment]
any other hint?

**Comment by hallvors** [comment]
LGTM

**Pull request merged**